File: AquariumApi.Models/Aquarium/WirelessDevice.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace AquariumApi.Models
{
    public class WirelessDevice
    {
        public int? Id { get; set; }
        public List<WirelessDeviceSensor> Sensors { get; set; }
        public string Name { get; set; }
        public string Hostname { get; set; }
        public DateTime LastConnected { get; set; }
        [NotMapped]
        public WirelessDeviceStatus Status { get; set; }
    }
}

File: AquariumApi.DeviceApi/ClientApp/src/app/modules/SharedDeviceModule/models/WirelessDevice.ts
import { WirelessDeviceStatus } from "./WirelessDeviceStatus";
import { WirelessDeviceSensor } from "./WirelessDeviceSensor";

export class WirelessDevice {
  id: number | null;
  hostname: string;
  sensors: WirelessDeviceSensor[]
  lastConnected: string;
  status: WirelessDeviceStatus
}


